charts/platform-operator/templates/tekton/kaniko-build-step-local.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kaniko-docker-build-step-local
+  namespace: kagenti-system  
+  labels:
+    kagenti.operator.dev/tekton: step 
+data:
+  task-spec.yaml: |
+    params:
+      - name: DOCKERFILE
+        type: string
+        description: Path to the Dockerfile relative to context
+        default: "Dockerfile"
+      - name: image
+        type: string
+        description: Destination image reference
+      - name: SKIP_TLS_VERIFY
+        type: string
+        description: Skip TLS verification when pushing to registry
+        default: "false"
+      - name: subfolder-path
+        type: string
+        description: Path to the subfolder to use as build context
+      - name: registry-secret
```

Comment:
do we still need to have this parameter if it's not used for local build?

Reply:
testing now